### ISSUE TITLE
Could not load file or assembly 'Neo.Lua, Version=5.3.0.0, ....

### ISSUE THREAD

**Issue by RobertBaruch** [Aerizeon/NeosWikiAssetGenerator#5]
```
2:25:09 PM.828 ( 60 FPS)	Exception running asynchronous task:
System.AggregateException: One or more errors occurred. ---> System.TypeLoadException: Could not set up parent class, due to: Could not load type of field 'NeosWikiAssetGenerator.Type_Processors.NeosTypeProcessor:LuaInstance' (4) due to: Could not load file or assembly 'Neo.Lua, Version=5.3.0.0, Culture=neutral, PublicKeyToken=fdb0cd4fe8a6e3b2' or one of its dependencies. assembly:Neo.Lua, Version=5.3.0.0, Culture=neutral, PublicKeyToken=fdb0cd4fe8a6e3b2 type:<unknown type> member:(null) signature:<none> assembly:F:\Steam\steamapps\common\NeosVR\Libraries\NeosWikiAssetGenerator.dll type:NeosTypeProcessor member:(null) signature:<none>
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine] (TStateMachine& stateMachine) [0x0002c] in <9577ac7a62ef43179789031239ba8798>:0 
  at NeosWikiAssetGenerator.WikiAssetGenerator.GenerateNew () [0x0001f] in <4b668905f0644cfa8a016fac336e4bf2>:0 
  at NeosWikiAssetGenerator.WikiAssetGenerator+<<GeneratePressed>b__10_0>d.MoveNext () [0x0000f] in <4b668905f0644cfa8a016fac336e4bf2>:0 
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.TypeLoadException: Could not set up parent class, due to: Could not load type of field 'NeosWikiAssetGenerator.Type_Processors.NeosTypeProcessor:LuaInstance' (4) due to: Could not load file or assembly 'Neo.Lua, Version=5.3.0.0, Culture=neutral, PublicKeyToken=fdb0cd4fe8a6e3b2' or one of its dependencies. assembly:Neo.Lua, Version=5.3.0.0, Culture=neutral, PublicKeyToken=fdb0cd4fe8a6e3b2 type:<unknown type> member:(null) signature:<none> assembly:F:\Steam\steamapps\common\NeosVR\Libraries\NeosWikiAssetGenerator.dll type:NeosTypeProcessor member:(null) signature:<none>
  at System.Runtime.CompilerServices.AsyncTaskMethodBuilder.Start[TStateMachine] (TStateMachine& stateMachine) [0x0002c] in <9577ac7a62ef43179789031239ba8798>:0 
  at NeosWikiAssetGenerator.WikiAssetGenerator.GenerateNew () [0x0001f] in <4b668905f0644cfa8a016fac336e4bf2>:0 
  at NeosWikiAssetGenerator.WikiAssetGenerator+<<GeneratePressed>b__10_0>d.MoveNext () [0x0000f] in <4b668905f0644cfa8a016fac336e4bf2>:0 <---


   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at void FrooxEngine.CoroutineManager.CheckExceptions(Task task)
   at void System.Threading.Tasks.ContinuationTaskFromTask.InnerInvoke()
   at void System.Threading.Tasks.Task.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem()
   at bool System.Threading.ThreadPoolWorkQueue.Dispatch()
   at bool System.Threading._ThreadPoolWaitCallback.PerformWaitCallback()
```

**Comment by RobertBaruch** [comment]
Copying `Neo.Lua.dll` from the nuget package dir (`C:\Users\xxx\.nuget\packages\neolua\1.3.13\lib\netstandard2.0\Neo.Lua.dll`) into `Neos_Data/Managed` solved that issue.

A bit hacky. Maybe it can go into the same directory as the plugin instead (Libraries)?

**Comment by Aerizeon** [comment]
I set it up to copy to the plugins directory as you suggested, and provided it as part of the launch options for neos.
This will put an error in the log (since it's not a neos plugin), but will still load it into memory where it can be used by the asset generator plugin.

Fixed in 4bf69f8c9d289eebf23e21bbb9d91827200a2ded

Thanks!
